### ISSUE TITLE
Gloomhaven: Jaws of the Lion: item 25 missing (barbed chain)

### ISSUE THREAD

**Issue by speakanton** [heisch/gloomhaven-item-db#21]
In the Gloomhaven item db for Jaws of the lion the item #25 "Barbed Chain" is missing. 

![image](https://user-images.githubusercontent.com/192207/189544878-0e057c39-9dd7-4b85-966c-7d6bef79fef0.png)

**Comment by solomojb** [comment]
Ohh good catch.  Thanks for reporting it. I’ve fixed it and pushed it up.
